en/includes/guides/authentication/sso-integrations/add-slack-template.md

```diff
@@ -0,0 +1,72 @@
+# Log in to Slack using the {{ product_name }}
+
+This page guides you through using {{ product_name }} to log in to Slack.
```

Comment:
```suggestion
This page guides you through setting up {{ product_name }} to add SSO (Single Sign-On) login to a Slack workspace.
```

Reply:
https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b